ISISComputingGroup/ibex_utils
# Ticket 2697: Script to gather all motor parameters

See https://github.com/ISISComputingGroup/IBEX/issues/2697

To test:
* Run genie_python pointing to an instrument with a mix of galils and other motors
* Import the script into genie_python
* Run get_params_and_save
* Confirm that motor parameters for all axes are in exported csv

Motion control group have already confirmed that columns are as expected.
